manual/sphinx/user_docs/bout_options.rst

```diff
@@ -619,6 +648,87 @@ has a method::
 
 This is currently quite rudimentary and needs improving.
 
+Reading and writing to NetCDF
+-----------------------------
+
+If NetCDF4 support is enabled, then the ``OptionsNetCDF`` class
+provides an experimental way to read and write options.
```

Comment:
Should we put it in a namespace like `bout::experimental::`? We could then feel free to change the API without deprecating things first

Reply:
Yes that sounds like a good plan